tools/AutoTuner/docker-compose.yml

```
@@ -0,0 +1,42 @@
+services:
+  # Ray Head Node (Server)
+  ray-head:
+    image: openroad/orfs:${ORFS_VERSION}
+    container_name: ray-head
+    command: >
+      sh -c "cd /OpenROAD-flow-scripts &&
+      pip install -r ./tools/AutoTuner/requirements.txt &&
+      ray start --head --dashboard-port=8265 --port=6379 --dashboard-host=0.0.0.0 --block"
```

```suggestion
      cd /OpenROAD-flow-scripts
      pip install -r ./tools/AutoTuner/requirements.txt
      ray start --head --dashboard-port=8265 --port=6379 --dashboard-host=0.0.0.0 --block
```

Is this a valid syntax?

---

Yes it is valid. Will resolve merge conflict for `test_helper.sh` when the other pending PR is merged, to avoid wasting runs

---

@luarss I think is safe to make the changes now.